pygmo/core.cpp

```diff
@@ -885,8 +892,80 @@ BOOST_PYTHON_MODULE(core)
 
     moead_.def("get_seed", &moead::get_seed);
 
+    // Exposition of various structured utilities
+    // Hypervolume class
+    bp::class_<hypervolume>("hypervolume", "Hypervolume class.", bp::no_init)
+        .def(bp::init<population, const bool>((bp::arg("pop"), bp::arg("verify") = true)))
+        .def("__init__",
+             bp::make_constructor(+[](const bp::object &points,
+                                      const bool verify) { return ::new hypervolume(pygmo::to_vvd(points), verify); },
+                                  bp::default_call_policies(), (bp::arg("points"), bp::arg("verify") = true)))
+        .def("compute", +[](hypervolume &hv, const bp::object &r_point) { return hv.compute(pygmo::to_vd(r_point)); },
```

Comment:
My point was that if, e.g., ``hv.compute()`` is a const method, I'd like for consistency the exposition wrapper to take ``const hypervolume &hv`` as first argument rather than ``hypervolume &hv``.